ui/app/src/components/PreviewConcierge/PreviewConcierge.tsx

```diff
@@ -500,6 +500,8 @@ export function PreviewConcierge(props: PropsWithChildren<{}>) {
     const localEditMode = getStoredEditModeChoice(user.username);
     if (nnou(localEditMode) && editMode !== localEditMode) {
       dispatch(setPreviewEditMode({ editMode: localEditMode }));
+    } else if (nou(localEditMode) && nnou(defaultEditMode) && editMode !== defaultEditMode) {
+      dispatch(setPreviewEditMode({ editMode: defaultEditMode }));
```

Comment:
I think the idea would be to have a `initPreviewConfig` similar to all other config sets and that would set things at the reducer level. The components never look at the _initial_ config value.